### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by RDIL**
Signed-off-by: Reece Dunham <me@rdil.rocks>

## PR Summary

Refactoring stuff, cleaning up the repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,7 +95,8 @@ Also make sure to check out our [PowerShell-RFC repository](https://github.com/p
 [known issues]: https://github.com/PowerShell/PowerShell/tree/master/docs/KNOWNISSUES.md
 [GitHub issues]: https://github.com/PowerShell/PowerShell/issues
 [new issue]:https://github.com/PowerShell/PowerShell/issues/new
-[development]: https://github.com/PowerShell/PowerShell-Docker/tree/master/docs/development.md
+[development]: https://github.com/PowerShell/PowerShell-Docker/blob/master/docs/development.md
```
Comment (TravisEz13) [inline README.md:98]: I don't see where you use this link

Comment (RDIL) [reply README.md:98]: Its used in the development and contributing section - this was just a dead link fix

**Comment by TravisEz13** [comment]
@PoshChan  Please remind me in 1 hour

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 1 hour ago

**Comment by TravisEz13** [comment]
archlinux and black arch have become unstable...

**Comment by RDIL** [comment]
alright I'll move them

**Pull request merged**